### PR TITLE
Only count the latest review from any particular user

### LINKED ISSUE #364: Bors says ":-1: Rejected by code reviews" even though the PR was approved
See https://github.com/ddnet/ddnet/pull/1079#issuecomment-372496230. Probably caused by #362.

### PR DESCRIPTION
**Pull request by notriddle**
Fixes #364

Regression introduced by #362

### CODE REVIEW

File: test/github_reviews_test.exs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "state" => "APPROVED",
+        "author_association" => "ADMIN"},
+      %{
+        "user" => %{"id" => 1},
+        "state" => "CHANGES_REQUESTED",
+        "author_association" => "ADMIN"}])
+    assert result == %{"APPROVED" => 0, "CHANGES_REQUESTED" => 1}
+  end
+
+  test "counts the last item (approval)", _ do
```
Comment (grahamc) [inline test/github_reviews_test.exs]: This test was the failing test prior?

**Comment by grahamc** [comment]
r+

Thank you, sorry :$

**Comment by grahamc** [comment]
d'oh

bors r+

File: test/github_reviews_test.exs (hunk 1 above)
Comment (notriddle) [reply test/github_reviews_test.exs]: Yes.

**Pull request merged**